andela/AndelaEatsUI
# #162038590 Fetch suspended vendors

#### What does this PR do?
This PR fetch and display all suspended vendors

#### Description of Task to be completed?
Fetch suspended vendors

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-admin-view-suspended-vendors-162038590`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as an `Admin`, navigate to `admin dashboard`, select `Vendor Icon`, on the sidebar and click on `Suspended Vendors`
- run the command `npm run test-dev` to ensure all tests are passing

#### What are the relevant pivotal tracker stories?
- [#story-id162038590](https://www.pivotaltracker.com/n/projects/2178749/stories/162038590)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1266" alt="screen shot 2018-11-20 at 1 28 46 am" src="https://user-images.githubusercontent.com/21138774/48743498-ab9e5d00-ec63-11e8-95a1-714eefe2d6f9.png">
